### ISSUE TITLE
What the envirment?

### ISSUE THREAD

**Issue by BlueSpace-ice** [AsmaBaccouche/Connected-Unets-and-more#1]
I'm sorry to bother you. I hope to know the environment of this project, or I can submit a requirements.txt for your reference. Thank you.

**Comment by AsmaBaccouche** [comment]
absl-py==1.0.0
aniso8601==9.0.1
astunparse==1.6.3
bcrypt==3.2.2
bidict==0.22.0
blinker==1.4
cachetools==5.0.0
certifi==2021.10.8
cffi==1.15.0
charset-normalizer==2.0.12
click==8.1.2
cycler==0.11.0
Flask==2.1.1
Flask-Cors==3.0.10
Flask-Mail==0.9.1
Flask-RESTful==0.3.9
Flask-SocketIO==5.1.1
Flask-SQLAlchemy==2.5.1
flatbuffers==2.0
fonttools==4.31.2
gast==0.5.3
google-auth==2.6.2
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
greenlet==1.1.2
grpcio==1.45.0
h5py==3.6.0
idna==3.3
imageio==2.16.1
importlib-metadata==4.11.3
itsdangerous==2.1.2
Jinja2==3.1.1
keras==2.8.0
Keras-Preprocessing==1.1.2
kiwisolver==1.4.0
libclang==13.0.0
Markdown==3.3.6
MarkupSafe==2.1.1
matplotlib==3.5.1
mysql-connector-python==8.0.28
networkx==2.7.1
numpy==1.22.3
oauthlib==3.2.0
opencv-python==4.5.5.64
opt-einsum==3.3.0
packaging==21.3
Pillow==9.0.1
protobuf==3.19.4
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
pydicom==2.2.2
PyJWT==2.4.0
PyMySQL==1.0.2
pyparsing==3.0.7
python-dateutil==2.8.2
python-dotenv==0.19.2
python-engineio==4.3.1
python-socketio==5.5.2
pytz==2022.1
PyWavelets==1.3.0
requests==2.27.1
requests-oauthlib==1.3.1
rsa==4.8
scikit-image==0.19.2
scipy==1.8.0
six==1.16.0
SQLAlchemy==1.4.35
tensorboard==2.8.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorflow==2.8.0
tensorflow-io-gcs-filesystem==0.24.0
termcolor==1.1.0
tf-estimator-nightly==2.8.0.dev2021122109
tifffile==2022.3.16
typing_extensions==4.1.1
urllib3==1.26.9
Werkzeug==2.0.3
wrapt==1.14.0
zipp==3.7.0
